GOVERNMENT.md

```diff
@@ -28,10 +28,9 @@ _"SNIRF Contributors"_ are individuals who have contributed changes, revisions,
 
 **Section 3.** Terms. The Steering Committee is composed of 3 or 5 members of the fNIRS community who are elected to serve 3 year terms. The number of members should remain odd.
 
-**Section 4.** Meetings. A majority of Steering Committee members must meet at least biannually. At least one Maintainer must be present at these meetings. These meetings should review the following:
-- The most recent Maintainers Report
+**Section 4.** Meetings. A majority of Steering Committee members must meet at least biannually (twice a year). At least one Maintainer must be present at these meetings. These meetings should review the following:
 - Any pending releases or critical changes
-- project priorities and progress towards goals
+- Project priorities and progress towards goals
 - Delegation of resources (i.e maintainers' attention) towards completion of goals
 
 The meeting should result in the authorship of a brief report.
```

Comment:
Did we also want to remove the requirements for the meeting reports?

Reply:
We make notes in Github (well @samuelpowell does). Maybe just rephrase that the meeting decisions shall be documented?